### PR TITLE
added docs for core funcs under others section

### PR DESCRIPTION
**Pull request by Zymrael**
Added descriptions for some functions in the **Others** section of fastai **core**

- even_mults
- func_args
- noop
- one_hot
- show_some

### CODE REVIEW

File: docs_src/core.ipynb [hunk 1]
```diff
@@ -2598,6 +2598,106 @@
     "show_doc(even_mults)"
```
Comment (sgugger) [inline docs_src/core.ipynb]: <p>I think you mean `func_args` here, no?</p>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/1710/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='124'/>

**Comment by sgugger** [comment]
Thanks a lot, those are great examples! I just noticed one typo. If you can fix it before I merge this that would be great (follow the link to ReviewNB to see which cell).

**Comment by Zymrael** [comment]
Fixed it! Glad you like them :)

**Comment by sgugger** [comment]
Don't hesitate to add more like this ;) Thanks again!

**Pull request merged**